cluster/gce/container-linux/configure-helper.sh

```diff
@@ -459,6 +460,8 @@ function start-kubelet {
   flags+=" --cluster-domain=${DNS_DOMAIN}"
   flags+=" --config=/etc/kubernetes/manifests"
   flags+=" --experimental-check-node-capabilities-before-mount=true"
+  flags+=" --require-kubeconfig"
```

Comment:
I think this will become unnecessary in 1.6, but is needed for 1.5 and is fine for now